screen/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        .board {
+            background-color: #edcef1;
+            height: 825px;
+            width: 1820px;
+            padding: 20px;
+            margin: 20px;
+        }
+
+        .hidden-text {
+            display: none;
+        }
+
+        .custom-font {
+            font-family: "Georgia", serif;
+            color: #594e5c;
+        }
+
+        .custom-button {
+            font-family: "Georgia", serif;
+            background-color: #edcef1;
+            color: #594e5c;
+            border: none;
+            text-align: left;
+            width: 430px;
+        }
+
+        .custom-button:hover {
+            background-color: #dbaee0;
+            color: #594e5c;
+            width: 434px;
+        }
+        .custom-textArea {
```

Comment:
```suggestion
        .custom-textarea {
```
attention au mix `kebab-case` + `camelCase`